core/main/src/broker/thunder_broker.rs

```diff
@@ -216,12 +227,7 @@ impl ThunderBroker {
                 .generate_state_change_subscribe_request();
             {
                 let mut ws_tx = ws_tx_wrap.lock().await;
-
-                let _feed = ws_tx
-                    .feed(tokio_tungstenite::tungstenite::Message::Text(
-                        status_request.to_string(),
-                    ))
-                    .await;
+                let _feed = ws_tx.feed(Message::Text(status_request.to_string())).await;
```

Comment:
The usage of the following paired functions are there at multiple places. 
```
ws_tx.feed 
ws_tx.flush 
```
Wrap this under a re-usable function.

Reply:
Enhancement for future.